package capkproto;

import "capk_globals.proto";

message order_cancel_reject {
    optional bytes		orig_cl_order_id = 1;
    optional bytes		cl_order_id = 2;
    optional bytes		strategy_id = 3;
    optional sfixed32	order_status = 4;
	optional sfixed32	cancel_reject_reason = 5;
	optional sfixed32	cancel_reject_response_to = 6;
	optional string		cancel_reject_text = 7;
}

